R/load.R

```diff
@@ -252,12 +372,19 @@ readDot <- function (input = NULL, string = NULL, silent = FALSE) {
     lines <- unlist(strsplit(string, "\n|\t|\r\n"))
   }
   paths <- lines[grepl("[<-][->]", lines)]
-  if (length(paths) == 0)
-    stop("Could not recognise the input contents as DOT formatted connections.", call. = FALSE)
+  if (length(paths) == 0) {
+    stop("Could not recognise the input contents as DOT formatted connections.", 
+         call. = FALSE)
+    }
   # if something looks like a badly formatted connector, complain and stop
-  if (any(grepl("<<|>>|>-|-<|><|<>|<->", paths)))
-    stop("The input appears to have badly formatted connectors ('<<', '>>', '>-', '-<'', '><', '<>' or '<->'). Please fix these before continuing.", call. = FALSE)
+  
+  if (any(grepl("<<|>>|>-|-<|><|<>|<->", paths))) {
+    stop("The input appears to have badly formatted connectors",
+         " ('<<', '>>', '>-', '-<'', '><', '<>' or '<->'). Please fix",
+         " these before continuing.", call. = FALSE)
+    }
   # there's probably a smarter way to do these, but hey, this works.
+  
   paths <- gsub("[ ]*<-", "<-", paths)
```

Comment:
```suggestion

  # there's probably a smarter way to do these, but hey, this works.
  paths <- gsub("[ ]*<-", "<-", paths)
```

good example of a space making things confusing. I thought the comment above related to the if statement. Put a space in between the if-statement and the comment, but not between the comment and the code it refers to. Also, I love this comment lol